source/release-notes/2.4.0.rst

```diff
@@ -0,0 +1,63 @@
+* :jirabug:`PMM-5102`: The new "Node Details" dashboard now displays data from the hardware monitoring sensors in hwmon The `new dashboard <https://www.percona.com/doc/percona-monitoring-and-management/2.x/dashboards/dashboard-node-temperature-details.rst>`_ is based on the ``hwmon`` collector data from the node_exporter. Please note that data may be unavailable for some nodes because of the configuration or virtualization parameters
+
+Improvements
+================================================================================
+
+* :jirabug:`PMM-5470`: Clickhouse query optimized for Query Analytics to improve its speed and reduce the load on the backend
+* :jirabug:`PMM-5448`: The default high and medium metrics resolutions were changed to 1-5-30 and 5-10-60 sec. To reduce the effect of this change on existing installations, systems having the "old" high resolution chosen on the PMM Settings page (5-5-60 sec.) will be automatically re-configured to the medium one during an upgrade. If the resolution was changed to some custom values via API, it will not be affected
+* :jirabug:`PMM-5531`: A healthcheck indicator was implemented for the PMM Server Docker image. It is based on the `Docker HEALTHCHECK <https://docs.docker.com/engine/reference/builder/#healthcheck>`_. This feature can be tested as follows::
+
+     $ docker ps
+     $ docker inspect -f {{.State.Health.Status}}
+     $ until [ "`docker inspect -f {{.State.Health.Status}} pmm-server`" == "healthy" ]; do sleep 1; done
+
+* :jirabug:`PMM-5489`: The "Total" line in all charts is now drawn with the same red color for better consistency
+* :jirabug:`PMM-5461`: Memory graphs on the node-related dashboards were adjusted to have fixed colors that are more distinguishable from each other
+* :jirabug:`PMM-5329`: Prometheus in PMM Server was updated to version 2.16.0. This update has brought several improvements. Among them are significantly reduced memory footprint of the loaded TSDB blocks, lower memory footprint for the compaction process (caused by the more balanced choice of what to buffer during compaction), and improved query performance for the queries that only touch the most recent 2h of data.
+* :jirabug:`PMM-5210`: Data Retention is now specified in days instead of seconds on the PMM Settings page. Please note this is the UI-only change, so the actual data retention precision is not changed
+* :jirabug:`PMM-5182`: The logs.zip archive available on the PMM Settings page now includes additional self-monitoring information in a separate "client" subfolder. This subfolder contains information collected on the PMM Server and is equivalent to the one collected on a node by the ``pmm-admin --summary`` command. 
+* :jirabug:`PMM-4915`: The Query Analytics dashboard now shows Time Metrics in the Profile Section as "AVG per query" instead of "AVG per second"
```

Comment:
it's better to  have this  at the top  of  "improvement" section, because important change